lamp/tasks/install_php.yml

```diff
@@ -1,32 +1,44 @@
 ---
+- set_fact:
+    php_version: 7.0
+  when: php_version is not defined
+
+- name: add php repo
+  become: yes
+  apt_repository:
+    repo: 'ppa:ondrej/php'
+    state: present
+    update_cache: yes
+  when: php_version != 7.0
+
 - name: install php
   become: yes
   apt:
-      name: "{{ item }}"
+      name: "php{{ '' if php_version == '7.0' else php_version }}-{{ item }}"
```

Comment:
No, I didn't have any issues. I read an article, that I conveniently can't find now, that talks about using the php version when installing via apt-get can cause issues.

If I can't figure out why I did this, I'll revert.